### PR TITLE
Update code gen verification

### PR DESCRIPTION
**Pull request by loosebazooka**
- Updates Makefile to actually generate all required files (-f only includes the last entry)
- Update github check action to make sure we are checking all code

Signed-off-by: Appu Goundan <appu@google.com>

### CODE REVIEW

File: .github/workflows/generate.yml [hunk 1]
```diff
@@ -36,9 +36,13 @@ jobs:
     steps:
     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3
 
-    - name: Compile protobuf
+    - name: Clear out protobuf directory
       run: |
-        make proto
+        rm -rf gen
```
Comment (znewman01) [inline .github/workflows/generate.yml:41]: nit: can we make this a `make clean` target?

**Review by znewman01 — COMMENTED** [review]
Does it actually make sense to commit the generated code?

We have to do this for Go because Go doesn't have support for generated code, whereas in Java you can generate it as part of the JAR or whatever.

We're going to want to publish this as a JAR too, right? (Okay if that's a different PR)

File: .github/workflows/generate.yml (hunk 1 above)
Comment (loosebazooka) [reply .github/workflows/generate.yml:41]: yeah I'm doing that... trying to figure out how to get docker to run as the user so clean will work with sudo on local machines

Comment (loosebazooka) [reply .github/workflows/generate.yml:41]: all the files are generated with ownership `root:root`

**Comment by loosebazooka** [comment]
> We're going to want to publish this as a JAR too, right? (Okay if that's a different PR)

Yeah that's a bit more involved. Will be in a followup

File: .github/workflows/generate.yml (hunk 1 above)
Comment (znewman01) [reply .github/workflows/generate.yml]: :shipit:

**Comment by loosebazooka** [comment]
> Does it actually make sense to commit the generated code?

soooo.. yes we can generate the java stuff on the fly with a gradle build file. Lemme just remove all the java stuff from here and keep the fixes to the go gen.

**Review by asraa — COMMENTED** [review]
Thanks for catching the `-f` bug!

**Review by znewman01 — DISMISSED** [review]

**Comment by loosebazooka** [comment]
@asraa  @znewman01 merge? I don't have permissions

**Comment by loosebazooka** [comment]
oh wait, need to update README

**Review by asraa — APPROVED** [review]
Thank you!

**Pull request merged**